TopMovies/View/MovieCollectionViewCell.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  MovieCollectionViewCell.swift
+//  TopMovies
+//
+//  Created by anikolaenko on 08.02.2021.
+//
+
+import UIKit
+import SnapKit
+import Nuke
+
+// MARK: - Props struct -
+struct MovieCollectionProps {
+    let adultLabelText: String
+    let ratingLabelText: String
+    let titleLabelText: String
+    let descriptionLabeltext: String
+    let posterURL: URL?
+    let posterPlaceholderImage: UIImage
+}
+
+// MARK: - Cell class -
+class MovieCollectionViewCell: UICollectionViewCell {
+    private let posterImageView = UIImageView()
+    private let adultLabel = UILabel()
+    private let ratingLabel = UILabel()
+    private let titleLabel = UILabel()
+    private let descriptionLabel = UILabel()
+    private let shortInfoStackView = UIStackView()
+    private let infoStackView = UIStackView()
+    // MARK: - Cell configuration
+    public func configureWith(props: MovieCollectionProps) {
+        posterImageView.image = props.posterPlaceholderImage
+        if let url = props.posterURL {
+            let options = ImageLoadingOptions(placeholder: props.posterPlaceholderImage,
+                                              transition: .fadeIn(duration: 0.2))
+            Nuke.loadImage(with: url,
+                           options: options,
+                           into: posterImageView) }
```

Comment:
Closing brace should be on the next line 😇

Reply:
:]

Reply:
👍